dashboard/src/App.js

```diff
@@ -1,26 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-//component imports
-import Index from "../src/components/Index";
+/*
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      items = []
+    }
+  }
+
+  render() {
+    const { items } = this.state;
+    return <>
+      <input> </input>
+      <h1> gg </h1>
+      <ul>
+        {items.map(item => <li>{item} </li>)}
+      </ul>
+    </>
+  }
+}*/
 
+//component imports
+import MainPage from "./components/MainPage";
+import Banner from "./components/Banner";
+import HowTo from "./components/HowTo";
+import Leaderboard from "./components/Leaderboard";
 //firebase imports
 
 //react + css
 
 //router 
 import {BrowserRouter as Router,Switch, Route } from "react-router-dom";
 
-function App() {
+const App = () => {
+
   return (
+    <div class="content">
     <Router>
+      <Banner></Banner>
       <Switch>
-        <Route path="/" exact component={Index} />
-        
+        <Route path="/" exact component={MainPage} />
+        <Route path="/howto" exact component={HowTo} />
+        <Route path="/leaderboard" exact component={Leaderboard} />
+        {/*
+        more routes
+        <Route path="/" exact component={} />
+        <Route path="/" exact component={} />
+        <Route path="/" exact component={} />
+        */}
```

Comment:
see above comment about comments. alternatively, add a TODO: comment so that you can search the repo for TODO: statements to figure out what work needs to be done